### PR TITLE
clean up desktop selection

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: package/skelcd-control-openSUSE.changes [hunk 1]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Aug 25 13:50:17 UTC 2016 - lnussel@suse.de
+
+- Remove LXDE from desktop selection
+- Remove Enlightenment from desktop selection
+- Remove Textmode
+- KDE is default so put it first
```
Comment (imobachgs) [inline package/skelcd-control-openSUSE.changes]: Please, add the version number here.

Comment (imobachgs) [inline package/skelcd-control-openSUSE.changes]: Never mind, it's in the other PR.

**Comment by imobachgs** [comment]
LGTM

**Comment by simotek** [comment]
Only just caught this, can someone give me a reason why Enlightenment was removed? It only appeared on the net install anyway.

**Comment by lnussel** [comment]
Exactly. I see no reason to differentiate there, nor why Enlightenment should have a privileged position . In fact I'd like to get rid of this 'Other' submenu rather sooner than later. The software selection screen can serve that purpose and it doesn't need to hardcode anything, just displays all available patterns.

**Comment by simotek** [comment]
To the contry this puts KDE, Gnome, and XFCE in privileged positions and leaves everything else dangling in no mans land.  The only reason Enlightenment is there and Mate / Cinnamon are not for example is that the work has not been put in to get them there (Mostly because last time they were not ready).

**Comment by tigerfoot** [comment]
I'm also not understanding why then only 3 desktop has to be on this screen. We always try to make openSUSE friendly with less known desktop env, because we believe in choice.
I also found it a bit unfair concerning the remove without discussing (which seems to be the case) with directly involved people, like simotek is for Enlightment.
Ludwig if you want to remove those desktop, then why not removing completely this intermediate stage, and rework the software selection with only patterns for n00b?

**Comment by simotek** [comment]
https://bugzilla.opensuse.org/show_bug.cgi?id=1003216 Has been created to cover the bug that selecting enlightenment no longer boots to enlightenment on first run. A previous version of openSUSE with a similar bug got some negative press.

**Pull request merged**